application/src/Api/Representation/MediaRepresentation.php

```diff
@@ -265,9 +265,10 @@ public function item()
      * Change the fallback title to be the media's source, if it exists.
      *
      * @param string|null $default
+     * @param string|null $lang
      * @return string|null
      */
-    public function displayTitle($default = null)
+    public function displayTitle($default = null, $lang = null)
```

Comment:
You should be passing this `$lang` here to the parent method call for it to have any effect.

Reply:
OK fixed in 52200f7